client/coral-admin/src/AppRouter.js

```diff
@@ -47,18 +51,19 @@ const routes = (
         <Route path='rejected' components={Moderation}>
           <Route path=':id' components={Moderation} />
         </Route>
-        <Route path='flagged' components={Moderation}>
+        <Route path='reported' components={Moderation}>
           <Route path=':id' components={Moderation} />
         </Route>
-        <Redirect from=':id' to='all/:id' />
+
+        <Route path=':id' components={Moderation} />
+
         <IndexRedirect to='all' />
```

Comment:
Also the other IndexRedirect should be removed as well on line 18: https://github.com/coralproject/talk/pull/813/files#diff-c7143cc58e326db8b86b6cdbb68e6170R18